lldb/include/lldb/Core/Module.h

```diff
@@ -818,6 +818,22 @@ class Module : public std::enable_shared_from_this<Module>,
   void ReportErrorIfModifyDetected(const char *format, ...)
       __attribute__((format(printf, 2, 3)));
 
+  void ReportWarningOptimization(
+      llvm::Optional<lldb::user_id_t> debugger_id = llvm::None);
+
+  void ReportWarningUnsupportedLanguage(
+      lldb::LanguageType language,
+      llvm::Optional<lldb::user_id_t> debugger_id = llvm::None);
+
+#ifdef LLDB_ENABLE_SWIFT
+  void ReportWarningCantLoadSwiftModule(
+      std::string details,
+      llvm::Optional<lldb::user_id_t> debugger_id = llvm::None);
+  void ReportWarningToolchainMismatch(
+      CompileUnit &comp_unit,
+      llvm::Optional<lldb::user_id_t> debugger_id = llvm::None);
```

Comment:
Repeating a question for [D123698](https://reviews.llvm.org/D123698), the use of `Optional` and a default is not used in this PR. Will this ever be called without a debugger_id?

Reply:
This was answered [here](https://reviews.llvm.org/D123698#inline-1185285)